src/graphql/schema.tsx

```diff
@@ -1,17 +1,15 @@
 import builder from './builder.tsx';
-import addDirectives from './lib/addDirectives.tsx';
 import './schemaImportMap.tsx';
 import { writeFileSync } from 'node:fs';
 import { join } from 'node:path';
-import { printSchemaWithDirectives } from '@graphql-tools/utils';
-import { lexicographicSortSchema } from 'graphql';
+import { printSchema } from 'graphql';
 
-const schema = addDirectives(builder.toSchema());
+const schema = builder.toSchema();
 
 if (process.env.NODE_ENV === 'development') {
   writeFileSync(
     join(process.cwd(), 'src/graphql/schema.graphql'),
-    printSchemaWithDirectives(lexicographicSortSchema(schema)),
+    printSchema(schema),
```

Comment:
schema is already sorted

Reply:
Oh that's great! Probably a leftover from the pre-pothos days.